cat/CaT-service.yaml

```diff
@@ -2597,24 +2670,91 @@ paths:
       security:
         - OAuth2:
             - buyer
-      summary: Create supplier further info request
-    put:
-      description: Respond to a Request for further information
+      summary: Create a message for an event
```

Comment:
Think this summary should be create an attachment for a message

Reply:
Made fix in latest version, thanks